pontoon/projects/forms.py

```diff
@@ -1,7 +1,18 @@
+import bleach
+
 from django import forms
 from django.core import validators
+from django.conf import settings
 
 
 class NotificationsForm(forms.Form):
     message = forms.CharField(widget=forms.Textarea)
     selected_locales = forms.CharField(validators=[validators.validate_comma_separated_integer_list])
+
+    def clean_message(self):
+        message = self.cleaned_data['message']
```

Comment:
Bleach can only clean, it doesn't have any exceptions (except missing settings). The only validation that we can implement is to make cleaned text and compare to user input and if the input do not match, then raise exception. Also bleach do not provide which not-allowed tags were user so we will need to find difference and use that for validation error message.